libraries/kubelet.rb

```diff
@@ -11,6 +11,7 @@ class KubeletService < Chef::Resource
     property :checksum, String,
       default: '4adaf40592248eef6fd4fa126464915e' +
                'a41e624a70dc77178089760ed235e341'
+    property :docker_service, String, default: 'docker.service'
```

Comment:
It's probably better to use `container_runtime_service` to be consistent with the kubelet nomenclature. http://kubernetes.io/docs/admin/kubelet/